static/src/stylesheets/module/site-messages/_first-pv-consent.scss

```diff
@@ -194,3 +194,65 @@ $btn-height: ($gs-baseline*3.5);
         }
     }
 }
+
+/** Classes added by CommercialConsentGlobalBanner **/
+
+.site-message--first-pv-consent--tallVariant {
+    @include mq(mobile) {
+        min-height: 45vh;
+    }
+    @include mq(phablet) {
+        min-height: 30vh;
+    }
+    @include mq(desktop) {
+        min-height: 20vh;
+    }
+    .site-message--first-pv-consent__block--head {
+        padding-top: $gs-baseline * 1.5;
+        padding-bottom: $gs-baseline * 1.5;
+    }
+    .site-message--first-pv-consent__block--intro {
+        padding-bottom: $gs-baseline;
+    }
+}
+
+.site-message--first-pv-consent--floatingVariant {
+    padding: $gs-baseline 0;
+    background-color: transparent;
+    .gs-container {
+        @include mq(mobile) {
+            width: 95%;
+        }
+        @include mq(phablet) {
+            width: 620px;
+        }
+        background-color: rgba($brightness-20, .98);
+        border-radius: $gs-baseline * .5;
+        margin: 0 auto $gs-gutter * .5;
+        padding-top: $gs-baseline * .5;
+        padding-bottom: $gs-baseline * 2;
+    }
+    .site-message__roundel {
+        display: none;
+    }
+    .site-message__copy {
+        padding-left: 0;
+        margin: 0 $gs-gutter;
+    }
+    .site-message--first-pv-consent__link {
+        margin-top: $gs-baseline * 1.5;
+        margin-left: 0;
+    }
+    .site-message--first-pv-consent__button {
+        margin-right: $gs-gutter * .5;
+    }
+
+.site-message--first-pv-consent--animationVariant {
+    transform: translateY(100%) translateZ(0); // translateZ to force GPU
```

Comment:
Might be worth checking if we see a change in the crash rate , especially on fronts. Just mentioning this to dotcom is probably enough?

Reply:
I'll ask, I'm aware of `translateZ(0)` being a problem when applying the rule to huge elements (thousands of px in size) or elements positioned thousands of pixels off screen, but this is a relatively small element, and I checked and found this technique applied in numerous places in `frontend`.